docker-compose.yml

```diff
@@ -1,28 +1,26 @@
-version: '2'
+version: "2"
 
 services:
   wamp:
     build: ./wamp
     environment:
-      - WAMP_NEAR_EXPLORER_PORT=8080
-      - WAMP_NEAR_EXPLORER_BACKEND_SECRET=back
+      - WAMP_GENERATED_SECRET=THIS_IS_LOCALHOST_SECRET
     ports:
-      - 8080:8080
+      - 10000:8080
 
   backend:
     build: ./backend
     environment:
-      - WAMP_NEAR_EXPLORER_URL=ws://wamp:8080/ws
-      - WAMP_NEAR_EXPLORER_BACKEND_SECRET=back
+      - WAMP_HOST=wamp
+      - WAMP_GENERATED_SECRET=THIS_IS_LOCALHOST_SECRET
     mem_limit: 1G
-    entrypoint: ["npm", "run", "start:testnet-with-indexer"]
+    entrypoint: ["npm", "run", "start:testnet"]
 
   frontend:
     build: ./frontend
     environment:
       - PORT=3000
-      - WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws
-      - WAMP_NEAR_EXPLORER_INTERNAL_URL=ws://wamp:8080/ws
+      - WAMP_HOST=localhost
     ports:
       - 3000:3000
-    command: ["npm", "run", "start:local-wamp-with-indexer-testnet"]
+    command: ["npm", "run", "start:local-wamp-testnet"]
```

Comment:
Given that the Docker build context is limited to `./backend` / `./frontend` folders, the `*.env` files from the root of the project won't be pulled in. I recommend we explicitly set the environment variables in `docker-compose.yml` file and use `start` and `start:local` commands.

Reply:
Where do we use docker images at the moment? Is it actually needed?
The easier solution would be to add the whole project directory to both backend / frontend projects.

Reply:
> The easier solution would be to add the whole project directory to both backend / frontend projects.

Please, don't.

As a rule of thumb, it is better to have the configuration options in the docker-compose.yml or pass it as [`--env-file`](https://docs.docker.com/compose/environment-variables/#using-the---env-file--option) rather then using dedicated commands like `-testnet`.

We use docker-compose as documentation, and it is also the easiest path to get Explorer running on a fresh machine.

Reply:
> As a rule of thumb, it is better to have the configuration options in the docker-compose.yml or pass it as [--env-file](https://docs.docker.com/compose/environment-variables/#using-the---env-file--option) rather then using dedicated commands like -testnet.

Agreed, I just used what happened to be before my changes.

> We use docker-compose as documentation, and it is also the easiest path to get Explorer running on a fresh machine.

Got it, I'll commit changes that will make docker work again tomorrow.